src/appdistribution/client.ts

```diff
@@ -22,7 +22,7 @@ export interface AppDistributionApp {
  * Proxies HTTPS requests to the App Distribution server backend.
  */
 export class AppDistributionClient {
-  static MAX_POLLING_RETRIES = 30;
+  static MAX_POLLING_RETRIES = 15;
```

Comment:
Forgot to mention this in the commit comment, but there's no good reason to poll 30 times...